### ISSUE TITLE
cli.js getting copied by default.

### ISSUE THREAD

**Issue by hemanth** [sindresorhus/generator-nm#20]
>? Do you need a CLI? No

But also I see `cli.js` I suspect [L#60](https://github.com/sindresorhus/generator-nm/blob/master/app/index.js#L60)

**Comment by floatdrop** [comment]
Dup. of #17 - can you provide steps to replicate this? Or more details of environment (OS, Node version)?

**Comment by hemanth** [comment]
```sh
❯ yo nm
? What do you want to name your module? temp
? What is your GitHub username? test
? What is the URL of your website? http://test.com
? Do you need a CLI? No
   create package.json
   create _package.json
   create editorconfig
   create gitattributes
   create gitignore
   create index.js
   create jshintrc
   create license
   create readme.md
   create test.js
   create travis.yml
   create .editorconfig
   create .gitattributes
   create .gitignore
   create .jshintrc
   create .travis.yml
```

```sh
$ uname -a
Darwin Laptop 14.1.0 Darwin Kernel Version 14.1.0: Mon Dec 22 23:10:38 PST 2014; root:xnu-2782.10.72~2/RELEASE_X86_64 x86_64
```
```
$ node -v
v2.3.4
```

**Comment by arthurvr** [comment]
Yep, I'm seeing this too it seems. Latest version of the generator. Thanks for reporting!

**Comment by floatdrop** [comment]
@arthurvr what version of `yo` installed?

**Comment by arthurvr** [comment]
1.4.7 (latest)

**Comment by hemanth** [comment]
^ same.

**Comment by gillstrom** [comment]
Me too. 
1.4.7, latest generator-nm and OS X.

**Comment by ragingwind** [comment]
Any progress?

**Comment by arthurvr** [comment]
> Any progress?

No, this simply depends on https://github.com/SBoudrias/mem-fs-editor/issues/45.

**Comment by SamVerschueren** [comment]
This issue was resolved in `node-glob@0.6.2`. It is fixed if you reinstall `generator-nm` because it pulls in that version.
